spec/rubocop/cop/performance/count_spec.rb

```diff
@@ -353,4 +353,17 @@ def count(&block)
       RUBY
     end
   end
+
+  context 'with `reject` with empty block body' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        array.reject {}.size
+              ^^^^^^^^^^^^^^ Use `count` instead of `reject...size`.
```

Comment:
Since empty block is not typically used, and the code produced by autocorrection would likely be non-idiomatic, so this case should probably be ignored by `expect_no_offenses`.

Reply:
Makes sense -- fixed.